packages/presentation/lib/screen/details/details_screen.dart

```diff
@@ -62,7 +62,9 @@ class _DetailsScreenState extends BlocScreenState<DetailsScreen, DetailsBloc> {
                     elevation: 0,
                     actions: [
                       IconButton(
-                        onPressed: () => {},
+                        onPressed: () => bloc.shareMovie(
+                            AppLocalizations.of(context)!
+                                .share(screenData?.movie?.traktId ?? 0)),
```

Comment:
1. move AppLocalizations.of(context)! to separate variable
2. rename shareMovie() to onShareButtonPressed and set return type as VoidCallback, then u can used just as:
onPressed: bloc.onShareButtonPressed()

Reply:
I don't think we can do it, since we need to pass the id param to method

Reply:
@sergei-mikhailovskii-idf is there any problem?
onPressed: bloc.onShareButtonPressed(l10n.share(screenData?.movie?.traktId ?? 0))

Reply:
my bad

Reply:
🤔

Reply:
confusing

Reply:
@sergei-mikhailovskii-idf no, my bad